core/unix/module_elf.c

```diff
@@ -1449,6 +1450,46 @@ tlsdesc_resolver(struct tlsdesc_t *arg)
 
 #endif /* !ANDROID */
 
+#ifndef ANDROID
+
+#    ifdef RISCV64
+static int
+glibc_riscv_hwprobe(void *pairs, uint64_t pair_count, uint64_t cpu_count, uint64_t *cpus,
+                    uint32_t flags)
+{
+    return -dynamorio_syscall(SYS_riscv_hwprobe, 5, pairs, pair_count, cpu_count, cpus,
```

Comment:
Could you explain why this is negative?

Reply:
https://github.com/bminor/glibc/blob/575de3d6663273b9d4d8fd483a9b45a971aac3af/sysdeps/unix/sysv/linux/riscv/hwprobe.c#L34-L35